ext/http/http_next.rs

```diff
@@ -1048,3 +1077,16 @@ impl Resource for UpgradeStream {
     self.cancel_handle.cancel();
   }
 }
+
+#[op]
+pub async fn op_raw_write_vectored(
+  state: Rc<RefCell<OpState>>,
+  rid: ResourceId,
+  buf1: JsBuffer,
+  buf2: JsBuffer,
+) -> Result<usize, AnyError> {
+  let resource: Rc<UpgradeStream> =
```

Comment:
If this is not an `UpgradeStream`, it should probably fall back to the standard write methods. I don't think we should assume that any two-chunk write is a websocket write.

Reply:
In JS, All `TCPSERVERWRAP` provider type resources are an UpgradeStream so this does not need a fallback.